includes/Core/Assets/Assets.php

```diff
@@ -769,30 +769,16 @@ private function get_inline_user_data() {
 	 * @return array The inline data to be output.
 	 */
 	private function get_inline_data() {
-		$cache        = new Cache();
 		$current_user = wp_get_current_user();
 		$site_url     = $this->context->get_reference_site_url();
 		$input        = $this->context->input();
 		$page         = $input->filter( INPUT_GET, 'page', FILTER_SANITIZE_STRING );
 
 		$admin_data = array(
-			'siteURL'          => esc_url_raw( $site_url ),
-			'siteName'         => get_bloginfo( 'name' ),
-			'siteUserID'       => md5( $site_url . $current_user->ID ),
-			'adminRoot'        => esc_url_raw( get_admin_url() . 'admin.php' ),
-			'assetsRoot'       => esc_url_raw( $this->context->url( 'dist/assets/' ) ),
-			'nojscache'        => current_user_can( 'manage_options' ) && null !== $input->filter( INPUT_GET, 'nojscache' ),
-			'datacache'        => ( current_user_can( 'manage_options' ) && null !== $input->filter( INPUT_GET, 'datacache' ) )
-				? json_encode( $cache->get_current_cache_data() ) // phpcs:ignore WordPress.WP.AlternativeFunctions.json_encode_json_encode
-				: false,
-			'timestamp'        => time(),
-			'currentScreen'    => is_admin() ? get_current_screen() : null,
-			'currentAdminPage' => ( is_admin() && $page ) ? sanitize_key( $page ) : null,
-			'resetSession'     => $input->filter( INPUT_GET, 'googlesitekit_reset_session', FILTER_VALIDATE_BOOLEAN ),
-			'reAuth'           => $input->filter( INPUT_GET, 'reAuth', FILTER_VALIDATE_BOOLEAN ),
-			'ampEnabled'       => (bool) $this->context->get_amp_mode(),
-			'ampMode'          => $this->context->get_amp_mode(),
-			'homeURL'          => $this->context->get_canonical_home_url(),
+			'siteURL'      => esc_url_raw( $site_url ),
+			'adminRoot'    => esc_url_raw( get_admin_url() . 'admin.php' ),
```

Comment:
This wasn't mentioned in the ACs but I think this can be removed as well as it isn't used anymore.

Reply:
Removed.